### PR TITLE
ignore rustwide from Cargo's workspace

### PR DESCRIPTION
**Pull request by pietroalbini**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -9,6 +9,10 @@ build = "build.rs"
 edition = "2018"
 
 [workspace]
+exclude = [
+    "ignored",
+    ".workspace",
+]
```
Comment (jyn514) [inline Cargo.toml:15]: Can we instead use `members =` so we don't have to worry about any other folders the user has?
```suggestion
members = ["metadata"]
```

**Comment by jyn514** [comment]
For posterity: this is fallout from #1000.

File: Cargo.toml (hunk 1 above)
Comment (pietroalbini) [reply Cargo.toml:15]: Dunno why, but members doesn't work.

Comment (jyn514) [reply Cargo.toml:15]: This is https://github.com/rust-lang/cargo/issues/5418, it seems like there's no way to override this 🤦

Comment (jyn514) [reply Cargo.toml:15]: Ok, in that case looks good once the docker workspace is ignored too.
```suggestion
exclude = [
    "ignored",
    ".workspace",
    ".rustwide-docker",
]
```

**Pull request merged**